### ISSUE TITLE
99% working

### ISSUE THREAD

**Issue by wolgix** [boermansjo/ESX_HUD#7]
Hi can i get some contact to you?
I got it 99% working, it save, load, but i have problem with scrolling, and with clotheshop.
I can share with you if you can help me with that :)

**Comment by HexagoneM** [comment]
Scrolling is not fixed by now. What exactly is the problem with it?

**Comment by XvenDeR** [comment]
Share your code ill fix scroll for you

**Comment by HexagoneM** [comment]
And I will fix the clotheshop for you. :)

**Comment by wolgix** [comment]
Pm me on email - Wolgix@gmail.com

**Comment by HexagoneM** [comment]
He will send you an mediafire link to something uploaded on the forums.
![image](https://user-images.githubusercontent.com/45739302/62640248-d6dbf000-b949-11e9-996d-a49fcd410c62.png)

**Comment by wolgix** [comment]
But the scrolling and shop is bugged in it.

**Comment by HexagoneM** [comment]
Close the issue. The owner of the repository is trying to make this script saving in ESX tables rather in outfit one.

**Comment by boermansjo** [comment]
Yes, my script will use the default esx.
Also the scrolling is working perfect with my version. So if you want it working, just check the code. Mostly html, CSS and JS

**Comment by HexagoneM** [comment]
@boermansjo we are with you. If u need help feel free to ask buddy.

**Comment by Duderocktv** [comment]
This will be a drag and drop replacement at one point right? Are you guys
all forking your changes and such? Lots of people watching this script lol

**Comment by boermansjo** [comment]
Yes this will be a drag and drop. It should replace esx_skin when finished.

**Comment by boermansjo** [comment]
For now with a small change in the esx_skin it all works. Skin is saving for both male and female characters. I still need some work to clean some stuff. Also checking for bits I can put in the config.
(Like sex change in menu or getting the sex chosen before from an identity script.)
I will keep this issue open just in case.
And yes @Duderocktv , a lot of people fork stuff but never help or share.

**Comment by boermansjo** [comment]
> This will be a drag and drop replacement at one point right? Are you guys all forking your changes and such? Lots of people watching this script lol

@Duderocktv  It should be drag and drop now. If you used this script, make sure to change the hud:load back to the original ```esx_skin:openSaveableMenu```. I left the ```hud:loadMenu``` just in case.
